content/en/docs/client-options.md

```diff
@@ -39,13 +39,13 @@ The Python [acme](https://github.com/certbot/certbot/tree/master/acme) module is
 
 {{< /clients >}}
 
-# Adding your client/project
+# Adding client/project
```

Comment:
I think this needs a connecting word. What about "Adding a client/project" instead of "your"?

Reply:
Thanks for reviewing! I fixed that point now.